src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Roundtrip.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using Xunit;
+
+namespace System.Formats.Tar.Tests
+{
+    public class TarFile_CreateFromDirectory_Roundtrip_Tests : TarTestsBase
+    {
+        [ConditionalTheory(typeof(MountHelper), nameof(MountHelper.CanCreateSymbolicLinks))]
+        [InlineData("subDirectory/file.txt", "./file.txt", "subDirectory")]
+        [InlineData("./file.txt", "../file.txt", "subDirectory")]
+        [InlineData("subDirectory1/file.txt", "../file.txt", "subDirectory1/subDirectory1.1")]
+        [InlineData("subDirectory1/subDirectory1.1/file.txt", "./file.txt", "subDirectory1/subDirectory1.1")]
+        [InlineData("./file.txt", "./file.txt", null)]
+        public void SymlinkRelativeTargets_InsideTheArchive_RoundtripsSuccessfully(string targetFilePath, string symlinkTargetPath, string subDirectory)
```

Comment:
Could the first argument of this test be omitted and replaced by `Path.Combine(subDirectory, symlinkTargetPath)`?